src/components/Card/Card.tsx

```diff
@@ -26,33 +64,84 @@ export default function Card(props: React.PropsWithChildren<CardProps>) {
     children,
     className,
     ctas,
-    footer,
-    heading,
     id,
-    image,
+    layout,
+    border,
+    padding,
     modifiers = [],
   } = props;
   const baseClass = "card";
 
+  let cardStyle;
+  let childHeading;
+  let childContent;
+  let childImage;
+  let childActions;
+  let headingCount = 0;
+  let contentCount = 0;
+  let imageCount = 0;
+  let actionCount = 0;
+
+  if (layout) {
+    modifiers.push(layout);
+  }
+
+  if (border) {
+    modifiers.push("with-border");
+  }
+
+  if (padding) {
+    cardStyle = {
+      padding: padding,
+    };
+  }
+
+  React.Children.map(children, (child: React.ReactElement) => {
+    if (child.type === CardHeading) {
+      childHeading = child;
+      headingCount++;
+    }
+
+    if (child.type === CardContent) {
+      childContent = child;
+      contentCount++;
+    }
+
+    if (child.type === CardImage) {
+      childImage = child;
+      imageCount++;
+    }
+
+    if (child.type === CardActions) {
+      childActions = child;
+      actionCount++;
+    }
+  });
+
   return (
-    <div className={bem(baseClass, modifiers, blockName, [className])} id={id}>
-      {heading && (
-        <div className={bem("heading", modifiers, baseClass)}>{heading}</div>
-      )}
-      <div className={bem("columns", modifiers, baseClass)}>
-        {image && (
-          <div className={bem("image", modifiers, baseClass)}>{image}</div>
-        )}
-        {children && (
-          <div className={bem("content", modifiers, baseClass)}>{children}</div>
-        )}
-        {ctas && (
-          <div className={bem("ctas", modifiers, baseClass)}>{ctas}</div>
-        )}
+    <div
```

Comment:
The structure of the new `Card` component should be different from the old one.  The new Card is more composable, allowing multiple `CardHeading` and `CardContent` child components that can be rearranged and ordered in any configuration.  Additioanlly, the `CardActions` child component should be stacked with the `CardHeading` and `CardContent` components.  For a visual representation of where we are heading with the new `Card` component, please refer to the [Article](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=15955%3A23193) and [Books](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Main?node-id=15955%3A23231) cards in the Figma file.